language/move-vm/integration-tests/src/tests/instantiation_tests.rs

```diff
@@ -208,7 +208,7 @@ fn test_runner(
     let (res, time) = run_with_module(&mut gas, entry_spec);
     let err = res.err().unwrap().major_status();
     println!("* {}: {}ms - Status: {:?}", test_name, time, err);
-    assert_eq!(err, StatusCode::OUT_OF_GAS, "Must finish OutOfGas");
+    // assert_eq!(err, StatusCode::OUT_OF_GAS, "Must finish OutOfGas");
```

Comment:
feel free to comment that out for now, thanks!

Reply:
Thank you